### DISCUSSION TITLE
local adb connection for all clients

### DISCUSSION THREAD

**Discussion by ytpk** [yume-chan/ya-webadb#467] [Q&A]
hi, I want to use this thing as an Android emulator for a very weak phone. can I somehow connect from my phone to the adb connection on my computer? that is, without using ws or the like, if, say, I raise this server on the same PC that is already connected to the desired adb

**Comment by yume-chan** [comment]
> I want to use this thing as an Android emulator for a very weak phone.

Do you mean you want to connect to an emulator running on a PC, from your Android device? 

> without using ws or the like

Browsers will block unsecure connections to other devices, that's why usually the WebSockify software is running on the same device. If you can add a HTTPS certificate to your WebSockify server on PC, it can work.
